roles/etcd/tasks/check_certs.yml

```diff
@@ -33,10 +33,29 @@
        ['{{ etcd_cert_dir }}/ca.pem',
        {% set all_etcd_hosts = groups['k8s-cluster']|union(groups['etcd'])|union(groups['calico-rr']|default([]))|unique|sort %}
        {% for host in all_etcd_hosts %}
-       '{{ etcd_cert_dir }}/node-{{ host }}-key.pem'
-       {% if not loop.last %}{{','}}{% endif %}
+         '{{ etcd_cert_dir }}/node-{{ host }}-key.pem'
```

Comment:
This doesn't actually create a valid list. No commas are present here between elements. That's why it fails in CI.

Reply:
Still has no success with CI :(